Userland/Libraries/LibGUI/DatePickerDialog.gml

```diff
@@ -7,7 +7,7 @@
         fixed_height: 20
         fill_with_background_color: true
         layout: @GUI::HorizontalBoxLayout {}
-        
+
```

Comment:
Please make these corrections in the commit the DatePickerDialog was added instead.